app/javascript/components/analytics/widgets/MonitoreeFlow.js

```diff
@@ -38,7 +38,7 @@ class MonitoreeFlow extends React.Component {
                 <tr>
                   <th className="py-0"></th>
                   {MONITOREE_FLOW_HEADERS.map((monitoreeFlowHeader, index) => (
-                    <th key={index}>{monitoreeFlowHeader}</th>
+                    <th key={index}>{monitoreeFlowHeader === 'Total' ? 'All-Time' : monitoreeFlowHeader}</th>
```

Comment:
The reason for handling it this way instead of changing 'Total' to 'All-Time' in all the code is because there's already saved analytics data in the db using 'Total' so a migration would be required to make that work. If that's preferred though, I can definitely do that

Reply:
upon second thought, maybe it would make sense to just change it because the data beforehand is incorrect anyways? this would just result in the incorrect numbers changing to 0 instead

Reply:
Let's skip the migration